nvtabular/ops/categorify.py

```diff
@@ -192,6 +195,13 @@ def __init__(
         self.cat_cache = cat_cache
         self.stat_name = "categories"
         self.encode_type = encode_type
+        self.search_sorted = search_sorted
+
+        self.isfreqthreshold = (
+            isinstance(self.freq_threshold, int) and self.freq_threshold > 0
+        ) or (isinstance(self.freq_threshold, dict))
+        if self.search_sorted and self.isfreqthreshold:
+            raise ValueError("cannot use search_sorted=True with freq_threshold > 0.")
```

Comment:
Can we simplify this? Like isn't just testing if ```freq_threshold``` evaluates to true enough here, without needing a bunch of isinstance checks?

```suggestion
        if self.search_sorted and self.freq_threshold:
            raise ValueError("cannot use search_sorted=True with freq_threshold > 0.")
```

Reply:
> Can we simplify this? Like isn't just testing if `freq_threshold` evaluates to true enough here, without needing a bunch of isinstance checks?

@benfred same as  above, that was due to an error I was getting when freq_threshold is a dictionary. simply putting `f self.search_sorted and self.freq_threshold:` did not work. I will test it again, just in case.